.github/workflows/docker.yml

```diff
@@ -19,6 +19,11 @@ jobs:
     permissions:
       contents: read
       packages: write
+      id-token: write # needed for signing the images with GitHub OIDC Token
```

Comment:
The `id-token` is only necessary for the `provenance` step.

Reply:
Fixed, in my first approach, I had added the provenance step to the same job.